tests/it/dom-snapshot.ie11.it.test.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const {describe, it, before, after} = require('mocha');
+const {expect} = require('chai');
+const {resolve} = require('path');
+const createTestCafe = require('testcafe');
+const testPath = resolve(__dirname, 'testcafe-tests/*.testcafe.js');
+const testServer = require('../util/testServer');
+
+describe('Eyes integration tests', () => {
+  let closeTestServer;
+  before(async () => {
+    const server = await testServer({port: 7272});
+    closeTestServer = server.close;
+  });
+
+  after(async () => {
+    await closeTestServer();
+  });
+
+  it('works', async () => {
+    let testcafe = null;
+    let failedCount;
+    await createTestCafe('localhost', 1337)
+      .then(tc => {
+        testcafe = tc;
+        const runner = testcafe.createRunner();
+        return runner
+          .src([testPath])
+          .browsers(['saucelabs:Internet Explorer@11.0:Windows 7'])
```

Comment:
Throws `Error: 'assign' is not a function`

I'm assuming the latest version of the SDK gets a similar result on IE11.

Reply:
But `assign` isn't support in IE11. Seems like a case that's not transpiled by TestCafe. I'll check.

Reply:
Confirmed the same behavior on `master`